pkgs/development/libraries/egl-wayland/default.nix

```
@@ -69,6 +65,13 @@ in stdenv.mkDerivation rec {
     wayland
   ];
 
+  postFixup = ''
+    # Doubled prefix in pc file after postbuild hook replaces includedir prefix variable with dev output path
+    substituteInPlace $dev/lib/pkgconfig/wayland-eglstream.pc \
+      --replace "includedir=$dev/$dev" "includedir=$dev" \
```

I think it might be a good idea to simplify that to:
```suggestion
      --replace "=$dev/$dev" "=$dev" \
```
which would catch further regressions automatically.

---

Good idea, added the change.